agimus-demos/Makefile

```diff
@@ -0,0 +1,22 @@
+# robotpkg Makefile for:	wip/agimus-vision
+# Created:			Long Ha Thuc on Tue, 18 Aug 2020
+#
+
+ORG=			agimus
+NAME=			agimus-demos
+VERSION=		1.1.0
+DOWNLOAD_LOC=   releases/download
+MASTER_SITE_AGIMUS=     https://github.com/agimus/
+DISTNAME=		${NAME}-${VERSION}
+
+MASTER_SITES=		${MASTER_SITE_AGIMUS:=${NAME}/${DOWNLOAD_LOC}/v${VERSION}/}
+MASTER_REPOSITORY=	${MASTER_REPOSITORY_GITHUB}${ORG}/${NAME}/${DOWNLOAD_LOC}/v${VERSION}
+LICENSE=		BSD-2-Clause
```

Comment:
In this field, you can only put files that are in the `licenses` directory of the main robotpkg repository. Here, you need `2-clause-bsd`